### PR TITLE
Premium barrier extra content

### PR DESCRIPTION
**Pull request by adamsilver**

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -13,5 +13,9 @@
     "bower_components",
     "test",
     "tests"
-  ]
+  ],
+  "devDependencies": {
+    "o-fonts": "^1.6.4",
+    "next-sass-setup": "Financial-Times/next-sass-setup#^5.5.4"
```
Comment (matthew-andrews) [inline bower.json]: think this is a dep not a devdep

File: demo/main.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
+"use strict";
+/*global __dirname*/
```
Comment (matthew-andrews) [inline demo/main.js]: /*jshint node:true*/ ?

File: Makefile [hunk 3]
```diff
@@ -1,4 +1,4 @@
-.PHONY: test
+.PHONY: test build
```
Comment (adamsilver) [inline Makefile]: Check that I don't need build here

File: bower.json [hunk 4]
```diff
@@ -13,5 +13,9 @@
     "bower_components",
     "test",
     "tests"
-  ]
+  ],
+  "devDependencies": {
+    "o-fonts": "^1.6.4",
```
Comment (adamsilver) [inline bower.json]: See if i can kill

**Comment by aintgoin2goa** [comment]
:+1:

File: demo/main.js (hunk 2 above)
Comment (adamsilver) [inline demo/main.js]: My editor doesn't seem to require it, make verify is passing and perhaps we could use a .jshintrc file?

**Comment by adamsilver** [comment]
Add make clean

File: demo/main.js (hunk 2 above)
Comment (matthew-andrews) [inline demo/main.js]: try `origami-build-tools verify`

**Pull request merged**